static/w17/oh-plus-plus/wk4-ohpp.md

```diff
@@ -135,4 +135,70 @@ nmap <leader>erc :e ~/.config/nvim/init.vim<CR>
 * [A laundry list of interesting
 configurations](https://sheerun.net/2014/03/21/how-to-boost-your-vim-productivity/)
 
+## 'emacs'
+
+Key points:
+* Emacs is very command based:
+   - Many of the commands you will learn are started with other commands and direct typing of
+     commands afterwards (etc M -x ...., C -x ...)
+* Emacs has a versatile list of modes:
+   - Emacs has a mode for most everything you could want to do, thought it will require some
+     research to understand the route you wnat to take getting there
+* Emacs has all of its tools built into the editor
+   - Editing any of the settings and managing packages and customization is all handled through
+     Emacs itself because it has built in GUI's for these aspects of the editor.
+
+# First thing to know
+
+* In order to acces the myriad of commands in Emacs, the notation that begins accessing them begins with either:
+  - 'C -...' which is the same as pressing Ctrl with another key (examples like C -a). There is
+     also the command 'C -x', which prompts for another command to come afterwards (example C -x
+     (C -s).
+  --and--
+  - 'M -...' which is the same as pressin Alt with a key.  There is also 'M -x' and his key is 
```

Comment:
s/pressin/pressing/

It's probably worth mentioning that the meta key can change depending on platform. For instance, on my mac emacs is set to `esc` by default.